popde/density_estimate.py

```diff
@@ -0,0 +1,92 @@
+# First script for iterative KDE
+from awkde import GaussianKDE  #we will need our own code 
```

Comment:
The comment is not correct, as we might use or write new or different code to produce KDEs. 
Also we should not assume that any KDE is going to use awkde. I would prefer to only import awkde inside class methods that use it

Reply:
You don't seem to have read my request not to import awkde at the start of the module (as not all of the classes will need to use it) but only inside classes/methods that need it.

Reply:
I have removed the comment and at the moment import awkde inside class method. Once I have KDE coded this import will be replace or I will modify this method in the class.